docs/source/api/model/core.rst

```diff
@@ -31,3 +31,4 @@ Graph visualization
 
    model_to_networkx
    model_to_graphviz
+   model_to_mermaid
```

Comment:
These are not sorted... what do you think?

Reply:
as you want